ledger/participant-state/kvutils/BUILD.bazel

```diff
@@ -157,7 +157,7 @@ client_server_build(
     testonly = True,  # only test targets can depend on this.
     client = "//ledger/ledger-api-test-tool",
     client_args = [
-        "--exclude=LotsOfPartiesIT,TransactionScaleIT,CommandDeduplicationIT",
+        "--exclude=LotsOfPartiesIT,TransactionScaleIT",
```

Comment:
Shouldn't need this any more, because these tests are already excluded by default. We can just delete the line.